example/l2fwd_simple/odp_l2fwd_simple.c

```diff
@@ -129,8 +133,8 @@ static int run_worker(void *arg ODP_UNUSED)
 			odp_packet_free_multi(&pkt_tbl[sent], tx_drops);
 	}
 
-	if (total_pkts < 10)
-		g_ret = -1;
+	if (total_pkts >= 10)
```

Comment:
I would take the opportunity to remove this altogether, what are we trying to accomplish here?

Reply:
return 0 return code if some packets were processed and mark test as passed.

Reply:
This is not needed, we should only exit with an error code if some error happened to the application, and it should not be an error that no packets were processed (i.e. we used null pktio as input).

The test has other means to decide whether it passed successfully or not.

Reply:
yep, bash script validates in and out pcap files.